src/helpers.js

```diff
@@ -0,0 +1,27 @@
+export function getCharCodes(string) {
+	const codes = new Uint8Array(string.length);
+
+	for (let i = 0, length = string.length; i < length; i++) {
+		codes[i] = string.charCodeAt(i) & 0xff;
```

Comment:
Since destination is a `Uint8Array`, `& 0xff` is no longer necessary here - the typed array will take care of the wraparound.

Might be worth checking if there are more unnecessary bitcasts like this in the codebase.